<!-- Copyright (C) 2019 Earth Rover Limited. All rights reserved. -->

<!-- Scouting system
  This file launches the localization system.
  Author: David Llevat Pamies (dllevat@edgebrain.io)
  Date: 05/2019
-->

<launch>

  <!-- Settings -->
  <arg name="mounting"    default="tractor" />

  <node pkg="earth_rover_localization" type="start_localization.sh" name="localization_init" args="$(arg mounting)"/>

</launch>
